src/main/java/user_interface/RecipeAppShell.java

```diff
@@ -1,4 +1,30 @@
 package user_interface;
 
+import java.util.Scanner;
+
+/*
+    provides a command line user interface for input/output
+ */
 public class RecipeAppShell {
+    final static String GREETING = "Hello! Welcome to recipe book\n" +
```

Comment:
Do you think we could do this without static variables? In lecture the prof(s) seemed pretty adamant on avoiding static final and using instance variables and methods instead (I asked about this scenario)